### DISCUSSION TITLE
Scoop rancher desktop - manifest file not found

### DISCUSSION THREAD

**Discussion by ghost** [ScoopInstaller/Scoop#5533] [Help and Questions] [answered]
Hi,

I am running into a problem when i run the command 'scoop install rancher-desktop' - it says cannot find manifest file. I can download scoop successfully. any idea what the problem could be?

**Comment by HUMORCE** [comment]
`scoop bucket list`, `scoop search rancher-desktop`

Leave results of them.

**Reply by ghost** [reply]
<img width="441" alt="rd" src="https://github.com/ScoopInstaller/Scoop/assets/136502493/c454bcc3-dc37-451c-bb5b-eb5577cc570b">

I ran the suggested commands and I got this. Am i doing something wrong?

**Reply by HUMORCE** [reply, marked as answer]
`rancher-desktop` is a manifest in Extras bucket, you should add extras bucket first: `scoop bucket add extras`

**Reply by ghost** [reply]
Thank you for all your help. it worked!
